README.md

```diff
@@ -287,11 +287,30 @@ Normal build is
 
     cmake .. -DCMAKE_TOOLCHAIN_FILE=~/mjbots/kodlab_mjbots_sdk/cmake/pi.cmake
 
+
+
 # Running Code
 To Run code, first scp the binary onto the pi, and then Run it as sudo using:
 
     sudo ./code_binary
 
+# Troubleshooting 
+If you encounter the below a pthread error during Cmake keep the below commands in the main repo directory CMakeLists.txt after the endif()
+
+    set(CMAKE_THREAD_LIBS_INIT "-lpthread")
+    set(CMAKE_HAVE_THREADS_LIBRARY 1)
+    set(CMAKE_USE_WIN32_THREADS_INIT 0)
+    set(CMAKE_USE_PTHREADS_INIT 1)
+    set(THREADS_PREFER_PTHREAD_FLAG ON)
+
+If you run into another pthread error while running the "make" command then you can run the following command in a terminal to fix it
```

Comment:
This isn't clear. Its probably better to suggest redoing the rsync. Also be explicit about what sort of errors you saw.

Reply:
As the rsync was slow, I had received the rpi files from @zacgong, which he got from rpi. That could be a reason for the broken syslink. This needs to be checked if it occurs on either of the pi's with just plain rsync and not with copying of the files.